object_tracker_0/requirements.txt

```diff
@@ -9,6 +9,7 @@ accelerate
 mlflow
 psutil # So that mlflow can log system metrics
 pynvml # So that mlflow can log GPU metrics
+ffmpeg-python  # To extract video recording time from mp4 file
```

Comment:
I didn't know ffmpeg had a python api - very handy!

Reply:
I was a bit skeptikal at first, I thought it might launch entire processes to answer small things, but it doens't look like it's doing that, it felt pretty swift on my computer